Doc/whatsnew/3.9.rst

```diff
@@ -270,6 +270,9 @@ inspect
 :attr:`inspect.BoundArguments.arguments` is changed from ``OrderedDict`` to regular
 dict.  (Contributed by Inada Naoki in :issue:`36350` and :issue:`39775`.)
 
+Add :func:`inspect.ismethodwrapper` for checking if the object is a method wrapper.
+(Contributed by Hakan Çelik in :issue:`29418`.)
+
```

Comment:
This will need to be moved the "What's New" doc for 3.10.

Apologies for the delay in reviewing this.

Reply:
I moved the news text to the current version